### PR TITLE
Add new quotes, partners, and supporters

### PR DESCRIPTION
**Pull request by ironng**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -326,7 +326,15 @@ class App extends Component {
                                     twitter="https://twitter.com/jpackel"
                                     linkedin="https://www.linkedin.com/in/johnpackel" />
                             </Col>
-                            <Col sm={12} md={4}>
+                            <Col sm={12} md={3}>
+                                <Profile
+                                    name="James Prestwich"
+                                    title="Storj Labs"
```
Comment (mhluongo) [inline src/App.jsx]: @frdwrd would "Integral, formerly Storj Labs" be preferred?

**Comment by mhluongo** [comment]
I figure we fix James's title and we're good to go

**Comment by frdwrd** [comment]
"Integral, formerly Storj Labs" is good. Sorry to be the blocker

**Comment by ironng** [comment]
Updated!

**Comment by mhluongo** [comment]
Thanks @ironng @frdwrd!

**Pull request merged**